contracts/zero-ex/src/migration.ts

```diff
@@ -90,58 +111,113 @@ export interface FullFeatures extends BootstrapFeatures {
     transformERC20: string;
     signatureValidator: string;
     metaTransactions: string;
+    limitOrders: string;
 }
 
 /**
- * Extra configuration options for a full migration of the Exchange Proxy.
+ * Artifacts to use when deploying full features.
  */
-export interface FullMigrationOpts {
-    transformerDeployer: string;
+export interface FullFeatureArtifacts extends BootstrapFeatureArtifacts {
+    tokenSpender: SimpleContractArtifact;
+    transformERC20: SimpleContractArtifact;
+    signatureValidator: SimpleContractArtifact;
+    metaTransactions: SimpleContractArtifact;
+    limitOrders: SimpleContractArtifact;
 }
 
+/**
+ * Configuration for deploying full features..
+ */
+export interface FullFeaturesDeployConfig {
+    zeroExAddress: string;
+    wethAddress: string;
+    stakingAddress: string;
+    protocolFeeMultiplier: number | BigNumber;
```

Comment:
tbh not a huge fan of all the `number | BigNumber` types we've got floating around (especially in asset-swapper)
```suggestion
    protocolFeeMultiplier: number;
```

Reply:
this is a `uint32` anyway